src/_gettsim_tests/test_data/full_taxes_and_transfers/2008/hh_id_1.yaml

```diff
@@ -344,5 +344,15 @@ inputs:
       - 0.0
       - 0.0
       - 0.0
+    sum_eink_ohne_kapital_eink_y:
```

Comment:
Without these additions, the tests fail, since `sum_eink_ohne_kapital_eink_y` and `abgelt_st_y_tu` are not defined in 2008, but other functions seem to rely on them (which may be a bug).

Reply:
That is a bug indeed. Fixed it below.

And we should improve the error message. It should list the functions that require some node. Could you open an issue for that, please?

Reply:
-> #640